lib/ui/compositing.dart

```diff
@@ -57,7 +57,7 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
   ///
   /// See [pop] for details about the operation stack.
   EngineLayer pushTransform(Float64List matrix4) {
-    _throwIfMatrix4IsInvalid(matrix4);
+    _matrix4IsValid(matrix4);
```

Comment:
`assert(_matrix4IsValid(matrix4))` just to make it clear (which works since the method also returns a bool)

Reply:
Done.